src/patternfly/demos/AboutModal/examples/about-modal-example.hbs

```diff
@@ -1,7 +1,7 @@
 {{#> about-modal}}
   {{#> backdrop}}
     {{#> bullseye}}
-      {{#> about-modal-box about-modal-box--attribute='aria-labelledby="about-modal-title"'}}
+      {{#> about-modal-box about-modal-box--attribute='aria-label="about-modal-box"'}}
```

Comment:
This isn't the right fix for this issue and should be reverted. 

The AboutModal example listed under Components has the correct solution for this: https://pf4.patternfly.org/components/AboutModalBox/examples/

When I look at that example, it has this showing as the html for the heading. In the demo, the `id="about-modal-title"` is missing and therefore throwing an error. This html is what I would expect to see in the demo as well.
```
  <div class="pf-c-about-modal-box__header">
    <h1 class="pf-c-title pf-m-4xl" id="about-modal-title">
      Product Name
    </h1>
  </div>
```